src/DataGrid/DataGrid.php

```diff
@@ -54,7 +54,7 @@ public function render(string $view = 'laravel-datagrid::datagrid')
     {
         $request = request();
 
-        if ($request->ajax()) {
+        if ($request->wantsJson()) {
```

Comment:
Good point, but I'd rather use `->expectsJson()`
Also we need to tweak tests for this to work, in \WdevRs\LaravelDatagrid\Tests\DataGridTest::createRequest,
we need to add: `$request->headers->set('accept', 'application/json');`